functions/src/tests/setup.ts

```diff
@@ -0,0 +1,20 @@
+import { TEST_PAYLOAD } from "./utils";
+import * as admin from "firebase-admin";
+import { createUniqueId, SHARE_LINK_FIRESTORE_COLLECTION } from "../utils";
+
+// Test setup run before tests.
+module.exports = async () => {
```

Comment:
`module.exports` is the "old" way of creating module exports (before TypeScript or recent EcmaScript versions).  You should be able to change this to `export default async () => {` I think.

Though personally I would use normal "function" syntax though and do: `export async function setup() {` 🤷

Reply:
Ah, thanks for this context. I was able to remove this and use beforeAll() instead, but this is good to know for the future 🙏